app/js/services.js

```diff
@@ -0,0 +1,57 @@
+floripaApp.factory('searchService', function () {
+    return {};
```

Comment:
I use this service to keep state between view changes - in this case the search history. Since services are created once and live for the entire lifecycle of an angular app it seemmed a good way. I could also have used $rootScope, but it looks like a bad approach to put stuff there. To be honest I don't know a better way to do it.